README.md

```diff
@@ -226,7 +224,7 @@ export default Ember.Route.extend({
   ajax: Ember.inject.service(),
   model() {
     return this.get('ajax').request('/user/me')
-      .catch(function(error){
+      .catch(function(error) {
         if (error instanceof UnauthorizedError) {
```

Comment:
I left that code snippet alone because it was providing an example of the `AjaxError` subtypes.

Perhaps the that entire code block should be removed to focus on the one below that uses the helper function?